features/admin.applications.v1/components/settings/sign-on-methods/script-based-flow/script-templates-side-panel.tsx

```diff
@@ -16,7 +16,8 @@
  * under the License.
  */
 
-import AdaptiveAuthTemplateInfoModal from "@wso2is/admin.authentication-flow-builder.v1/components/predefined-flows-side-panel/adaptive-auth-template-info-modal";
+import AdaptiveAuthTemplateInfoModal from
```

Comment:
When we remove the index files and start importing from the file where the entity is defined itself, one challenge we would face is not being able to keep the line length under 120 characters.

One way we can handle this is to reduce the no. of nested folders and simplify the file organization so that the import statement length also gets shorter.

@brionmario any other suggestions to handle this?

Reply:
Applied a workaround [4b136f3](https://github.com/wso2/identity-apps/pull/7204/commits/4b136f3c03552d001c93280656fe8db01e9fb8c9)